src/Http/Routing/test/UnitTests/LinkGeneratorIntegrationTest.cs

```diff
@@ -0,0 +1,610 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Routing.Patterns;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Routing
```

Comment:
Add tests for using the route name + conventional route? https://github.com/aspnet/AspNetCore/issues/4468

Reply:
Similar https://github.com/aspnet/AspNetCore/issues/4556#issuecomment-448496662

Reply:
I'll add a test for a conventional route that doesn't match any endpoints.